### PR TITLE
staking options

### PR DESCRIPTION
**Pull request by kcper**
### All Submissions

* [x] Can you post a screenshot of your changes (if applicable)?
image
image

### New Feature Submissions

* [x] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?
#889 #888 
### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Added staking options for 90/180 days distribution time for SKILL/KING.
Added CBK Land staking options for each tier.

### CODE REVIEW

File: frontend/src/components/StakeSelectorItem.vue [hunk 1]
```diff
@@ -24,10 +24,11 @@
         </tr>
         <tr v-if="estimatedYield" :title="$t('stake.StakeSelectorItem.yieldTooltip')">
           <th class="bold">
-            {{$t('stake.StakeSelectorItem.apy')}}
+            {{ isNftStaking ? $t('stake.StakeSelectorItem.yield') : $t('stake.StakeSelectorItem.apy') }}
           </th>
           <td class="align-right">
-            {{ estimatedYield.multipliedBy(100).toFixed(2) }}%
+            {{ isNftStaking ? estimatedYield.toFixed(2) : estimatedYield.multipliedBy(100).toFixed(2) }}
+            {{ isNftStaking ? '/NFT' : '%' }}
```
Comment (Chriserus) [inline frontend/src/components/StakeSelectorItem.vue:31]: I would opt for parent div (in  this case td) to be rendered based on isNftStaking condition, to not check the same thing twice. But it's minor, do it, or not, we're good :)

File: frontend/src/components/smart/Staking.vue [hunk 2]
```diff
@@ -46,8 +46,9 @@
       <div class="stakePage medium-dark-bg">
         <div class="sPElement input">
           <div class="inputBody">
-            <div class="flex_row">
+            <div v-bind:class="isNftStaking ? 'd-flex align-items-center w-100' : 'flex_row'">
```
Comment (Chriserus) [inline frontend/src/components/smart/Staking.vue:49]: :class. no need to add "v-bind" part

File: frontend/src/components/smart/Staking.vue [hunk 3]
```diff
@@ -62,9 +63,20 @@
...
+              <div v-if="isNftStaking" class="mr-2 ml-3">ID:</div>
+              <b-form-select
+                class="nft-id-select"
+                v-if="isNftStaking"
+                :title="$t('stake.tokenAmount')"
+                v-model="idToStake">
+                <template #first>
+                  <b-form-select-option :value="null" disabled>Pick ID</b-form-select-option>
+                </template>
+                <option v-for="x in (isDeposit ? ownedLandIds : stakedIds)" :value="x.id" :key="x.id">{{x.id}} (Tier {{x.tier}})</option>
```
Comment (Chriserus) [inline frontend/src/components/smart/Staking.vue:75]: "Tier" can be added to translations

File: frontend/src/components/smart/Staking.vue [hunk 4]
```diff
@@ -62,9 +63,20 @@
...
               />
-              <div class="ant-col">{{ stakingTokenName }}</div>
+              <div v-if="isNftStaking" class="mr-2 ml-3">ID:</div>
+              <b-form-select
+                class="nft-id-select"
+                v-if="isNftStaking"
+                :title="$t('stake.tokenAmount')"
+                v-model="idToStake">
+                <template #first>
+                  <b-form-select-option :value="null" disabled>Pick ID</b-form-select-option>
```
Comment (Chriserus) [inline frontend/src/components/smart/Staking.vue:73]: Missing translation

File: frontend/src/components/smart/Staking.vue [hunk 5]
```diff
@@ -482,6 +514,10 @@ export default {
             await this.unstake({ amount, stakeType: this.stakeType });
           }
         }
+        if(isNftStakeType(this.stakeType)) {
+          await this.updateOwnedLands();
+          this.idToStake = null;
```
Comment (Chriserus) [inline frontend/src/components/smart/Staking.vue:519]: Why are we using nulls as initial values? In my opinion it should be undefined.

File: frontend/src/components/smart/Staking.vue [hunk 6]
```diff
@@ -849,4 +891,8 @@ export default {
   color: #9e8a57;
 }
 
+.nft-id-select {
+  width: 100%;
```
Comment (Chriserus) [inline frontend/src/components/smart/Staking.vue:895]: Delete custom class, you can use w-100

File: frontend/src/locales/en.json [hunk 7]
```diff
@@ -566,6 +566,7 @@
       "earn" : "Earn",
       "yieldTooltip" : "Estimated yield per year and token.",
       "apy" : "APY",
+      "yield": "YIELD",
```
Comment (Chriserus) [inline frontend/src/locales/en.json:569]: Try to avoid uppercase text in translations, use css uppercase

**Review by bilbolPrime — CHANGES_REQUESTED** [review]

**Review by bilbolPrime — APPROVED** [review]

**Review by Chriserus — APPROVED** [review]

**Pull request merged**